src/components/AlphabetFilter/alphabetFilterChangelogData.ts

```diff
@@ -14,7 +14,10 @@ export const changelogData: ChangelogData[] = [
     version: "Prerelease",
     type: "Update",
     affects: ["Functionality"],
-    notes: ["Removed explicit `classname` prop."],
+    notes: [
+      "Removed explicit `classname` prop.",
+      "Updates the internal use of ids to reduce `undefined` ids in the DOM.",
```

Comment:
Nit: would be nice to be consistent with the tense, but I don't really have a preference on past or present